spec/cb/clients/job_spec.rb

```diff
@@ -12,108 +12,6 @@
 
 module Cb
   describe Cb::Clients::Job do
-    describe '#search' do
```

Comment:
Actually, it looks like I should have deleted the code in job.rb.  Investigating a little further.